### PR TITLE
update chain 3333 from "Web3Q Testnet" to "EthStorage Testnet"

### PR DESCRIPTION
**Pull request by ping-ke**

### CODE REVIEW

File: _data/chains/eip155-3333.json [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
-    "name": "Web3Q",
-    "symbol": "W3Q",
+    "name": "Ether",
+    "symbol": "ETH",
     "decimals": 18
   },
-  "infoURL": "https://testnet.web3q.io/home.w3q/",
-  "shortName": "w3q-t",
+  "infoURL": "https://ethstorage.io/",
+  "shortName": "es-t",
```
Comment (qzhodl) [inline _data/chains/eip155-3333.json:12]: maybe we want to make sure there is no other chain whose short name is es and es-t

Comment (ping-ke) [reply _data/chains/eip155-3333.json:12]: No short name use es or es-t.

File: _data/chains/eip155-3333.json [hunk 2]
```diff
@@ -1,23 +1,16 @@
 {
-  "name": "Web3Q Testnet",
-  "chain": "Web3Q",
-  "rpc": ["https://testnet.web3q.io:8545"],
+  "name": "EthStorage Testnet",
+  "chain": "EthStorage",
+  "rpc": ["http://testnet.ethstorage.io:9540"],
```
Comment (syntrust) [inline _data/chains/eip155-3333.json:4]: Does the domain name testnet.ethstorage.io work?

Comment (qzhodl) [reply _data/chains/eip155-3333.json:4]: Yes, http://testnet.ethstorage.io has mapped to 65.108.236.27

**Review by qzhodl — APPROVED** [review]

**Review by syntrust — APPROVED** [review]

**Pull request merged**